tests/trainer/test_trainer.py

```diff
@@ -1180,81 +1180,75 @@ def val_dataloader(self):
     [
         (
             dict(accelerator=None, gpus=None),
-            dict(_strategy_type=None, _device_type=_AcceleratorType.CPU, num_gpus=0, num_processes=1),
+            dict(_strategy_type="single_device", _device_type=_AcceleratorType.CPU, num_gpus=0),
```

Comment:
Reviewers: In the previous accelerator_connector, `_strategy_type` is the intermediate stage, which doesn't exist in this version anymore. Here the current `_strategy_type` is the final strategy selection. This is not a fundamental logic change, and the strategic choice hasn't changed.